packages/flyout/src/Flyout.js

```diff
@@ -45,12 +47,15 @@ class Flyout extends Component<Props> {
   }
 
   toggleIsOpen = () => {
+    const {onStateChange } = this.props;
```

Comment:
Kan je je linter eens laten lopen en alle spaces en andere dingen waarover hij klaagt aanpassen. Alvast bedankt!

Reply:
Met de default config dan? er zit geen linting config in het project :)

Reply:
Oei, echt?! Ahahah…
Merci in ieder geval voor de wijzigingen!